### PR TITLE
Morg/chat avatars

### PR DESCRIPTION
**Pull request by morganecf**
https://github.com/morganecf/jitsi-party/issues/387 

This makes sure the converse chat stream user avatars are the same as the ones in the house. 
Screen Shot 2021-05-24 at 2 06 08 PM'

Note that I set the following CORS policy on the tstvhq bucket so that I could test it from `localhost`. Not sure if we want to get rid of it now? 
Screen Shot 2021-05-24 at 2 08 00 PM

### CODE REVIEW

File: app/client/jsx/JitsiVideo.jsx [hunk 1]
```diff
@@ -140,7 +140,7 @@ class JitsiVideo extends Component {
                 this.makeJitsiTransparent()
                 this.hideSpinner()
                 const { type, color } = this.props.jitsiData.avatar
-                const avatarUrl = Config.baseUrl + Config.avatars[type].images[color]
+                const avatarUrl = Config.avatars[type].images[color]
```
Comment (morganecf) [inline app/client/jsx/JitsiVideo.jsx:143]: This also fixes the avatar display in regular jitsi rooms.

**Comment by rene-grigory** [comment]
We should probably consider allowing the background color to be altered allowing the png to show better. I don't see where Converse.js allows this (it appears to default to a white background). We could override the style like so:
```
.avatar {
  background-color: <whatever color> !important;
}
```

**Comment by morganecf** [comment]
@rene-grigory  this doesn't look amazing but I created a variable for the avatar background color and backfilled it in the latest two themes with the same color we use in the sign in page. 

Screen Shot 2021-05-25 at 4 51 31 PM

**Review by hndrewaall — APPROVED** [review]
Thank you so much!

**Pull request merged**